yarn.lock

```diff
@@ -11904,6 +11904,17 @@ glob@^10.0.0, glob@^10.2.2:
     minipass "^7.0.4"
     path-scurry "^1.10.2"
 
+glob@^10.3.7:
```

Comment:
How come this file has been updated?

Reply:
this happens occasionally - generally its due to a yarn.lock being committed to master that doesn't include the submodule dependencies (due to the way NX/Lerna works these are all piled into one file)